packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -1,9 +1,20 @@
 <Project>
   <PropertyGroup>
     <_MicrosoftNetSdkWindowsDesktop>true</_MicrosoftNetSdkWindowsDesktop>
+   
     <EnableDefaultPageItems Condition="'$(EnableDefaultPageItems)' == ''">true</EnableDefaultPageItems> 
     <EnableDefaultApplicationDefinition Condition="'$(EnableDefaultApplicationDefinition)' == ''">true</EnableDefaultApplicationDefinition>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <_InvalidTargetFrameworkVersion>0.0</_InvalidTargetFrameworkVersion>  
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <_WindowsDesktopTargetFrameworkVersion Remove="@(_WindowsDesktopTargetFrameworkVersion)" />
+    <_WindowsDesktopTargetFrameworkVersion Include="$([MSBuild]::ValueOrDefault($(_TargetFrameworkVersionWithoutV), $(_InvalidTargetFrameworkVersion)))" />
```

Comment:
Why is this becoming an item? Is it just to get later evaluation? I'd prefer the define-the-property-in-targets approach.